### PR TITLE
Update TSGs

### PR DESCRIPTION
**Pull request by sridmad**

### CODE REVIEW

File: Cluster/Collecting logs for failed VMs.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+4. Copy the file out of the VM
+
+#### Mitigation
+##### Config issues
+- Check failures in the logs to eliminate any config issues
+    - For example, a reference to a resource could be wrong or no longer valid
+- Fix the config issue in the ARM template and redeploy it
+
+##### Other issues
+- Open a CSS case
```
Comment (sedwardstx) [inline Cluster/Collecting logs for failed VMs.md:30]: Open a Support Incident through the Azure Portal

Comment (inputoutputcode) [reply Cluster/Collecting logs for failed VMs.md:30]: Path/Wording is "Help + support", then "New support request"

File: Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md [hunk 2]
```diff
@@ -1,37 +1,68 @@
...
+
+- Validate that the configuration of failed extensions is correct
+- Service Fabric extension in error, note down the error message if any
+- See [Collecting logs for failed VMs](./Collecting%20logs%20for%20failed%20VMs.md) for obtaining more details about the failure and mitigation
+
+#### VMs healthy
+
+- Service Fabric may be failing to start up on the VM due to various issues
+- RDP to one or more VMs
+- Identify the list of running processes that match Fabric*.exe (in taskmanager)
```
Comment (inputoutputcode) [inline Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md:40]: Link to https://github.com/Azure/Service-Fabric-Troubleshooting-Guides/blob/master/Cluster/SF%20process%20list%20with%20Security%20Context%20info.md

File: Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md [hunk 3]
```diff
@@ -1,37 +1,68 @@
...
+  - Are any of them restarting (changing PID)
+    - If Fabric.exe is running and not restarting may indicate the VM is fine. Collect rest of the information from the VM and repeat the process for other VMs.
```
Comment (inputoutputcode) [inline Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md:42]: Proposal: rest of the listed information below

File: Cluster/Connecting to secure clusters with PowerShell.md [hunk 4]
```diff
@@ -39,17 +38,19 @@ $ClusterName= "{yourclustername}.{region}.cloudapp.azure.com:19000"
 
 #single command - compact example
 $connectArgs = @{  ConnectionEndpoint = $ClusterName;   }
-Connect-ServiceFabricCluster @connectArgs 
+Connect-ServiceFabricCluster @connectArgs
 ```
 
 Simple AAD Authentication
+
 ```PowerShell
```
Comment (inputoutputcode) [inline Cluster/Connecting to secure clusters with PowerShell.md:46]: The language name for code snippets in mark down should be in lower case. But looks like there is no official documentation and it is flexible interpreted.

File: Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md (hunk 2 above)
Comment (sridmad) [reply Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md:40]: Thanks for the link. We are just interested in the list of processes that match Fabric*.exe. Pointing to the other doc may add unnecessary complexity.

File: Cluster/Connecting to secure clusters with PowerShell.md (hunk 4 above)
Comment (sridmad) [reply Cluster/Connecting to secure clusters with PowerShell.md:46]: I am fixing any changes proposed by a VSCode plugin to the MD file to maintain uniformity. I will check if there are any warnings related to casing

File: Cluster/Collecting logs for failed VMs.md (hunk 1 above)
Comment (sridmad) [reply Cluster/Collecting logs for failed VMs.md]: I have left is generic as the path/wording may change in the future

File: Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md (hunk 3 above)
Comment (sridmad) [reply Cluster/Cluster Not Reachable  UpgradeServiceNotreachable.md]: fixed

File: Cluster/Connecting to secure clusters with PowerShell.md (hunk 4 above)
Comment (sridmad) [reply Cluster/Connecting to secure clusters with PowerShell.md]: fixed

**Pull request merged**